socketio/src/packet.rs

```diff
@@ -21,7 +21,7 @@ pub enum PacketId {
 pub struct Packet {
     pub packet_type: PacketId,
     pub nsp: String,
-    pub data: Option<String>,
+    pub data: Option<Value>,
```

Comment:
blocker: This is a public type, changing this would be a breaking change. While I agree data: Option<Value> makes the internal implementation more elegant, not worth introducing more breaking changes. Changing the payload type on a PATCH IMO is already pushing the boundary of what should be done on a PATCH even in 0.X.X
```suggestion
    // FIXME: on next breaking change consider changing this to a Value
    pub data: Option<String>,
```

Reply:
Since the `Payload::Multi` is a breaking change, so I change data to `Value`.